test/functional/helpers/constants/webContextConfig.js

```diff
@@ -0,0 +1,7 @@
+import baseConfig from "./baseConfig";
+
```

Comment:
I'm not sure we need those helpers since this is the only test that will use them. You can inline the config in the spec instead; up to you.